spark_pipeline_framework/utilities/fhir_helpers/fhir_sender_helpers.py

```diff
@@ -82,6 +86,7 @@ def send_fhir_delete(
     auth_scopes: Optional[List[str]],
     auth_access_token: Optional[str],
     log_level: Optional[str],
+    additional_request_headers: Optional[Dict[str, str]] = None,
 ) -> Dict[str, Any]:
     fhir_client: FhirClient = get_fhir_client(
         logger=logger,
```

Comment:
check if this logger will work because fhir client expect FHIRLogger object not getLogger object.